apps/src/maze/results/utils.js

```diff
@@ -0,0 +1,34 @@
+const Farmer = require('../farmer');
+const Bee = require('../bee');
+const Collector = require('../collector');
+const Wordsearch = require('../wordsearch');
+const Harvester = require('../harvester');
+const Planter = require('../planter');
+
+const FarmerHandler = require('./farmer');
+const BeeHandler = require('./bee');
+const CollectorHandler = require('./collector');
+const WordsearchHandler = require('./wordsearch');
+const HarvesterHandler = require('./harvester');
+const PlanterHandler = require('./planter');
+const ResultsHandler = require('./resultsHandler');
+
+export function createResultsHandlerForSubtype(controller, config) {
```

Comment:
I'm not a huge fan of this helper but I can't quite put my finger on why.  Can we use a hash of the subtype name to the appropriate handler, and look up by `controller.subtype.constructor.name`?

Will the extracted repo include `maze/results/*`?

Reply:
Sure!

And (as answered in another comment) no it will not